style/tailwind.css

```diff
@@ -1,4 +1,267 @@
-
 @import "tailwindcss/base";
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
+/** tailwind over-rides */
+
+h1,
+h2 {
+  font-size: xx-large;
```

Comment:
This should be xxx-large and h3 xx-large, h4 x-large please :).

Reply:
Okay dai, the font looked small. I will change it.